Extension/src/extensionActivationInformation.ts

```diff
@@ -0,0 +1,44 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All Rights Reserved.
+ * See 'LICENSE' in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+export enum InstallBlobStage {
```

Comment:
minor: I would prefer "InstallationStage" over "InstallBlobStage".  "InstallBlob" was a quick name that wasn't meant to be cannonized into a "thing" :)